public/locales/en/ironbank.json

```diff
@@ -7,11 +13,5 @@
   "no-ironbank-card": {
     "header": "No Iron Bank yet on {{network}}",
     "text": "Check back soon."
-  },
-  "ironbank-announcement-card": {
-    "header": "To our Iron Bank users",
-    "desc-1": "The Iron Bank will now operate as an independent project with its own communication channels and frontend. You will find your funds safe and available through their UI [app.ib.xyz](https://app.ib.xyz).",
-    "desc-2": "This decision provides the clarity that our communities need while enabling both parties to make decisions and move forward more quickly to build the future of DeFi.",
-    "desc-3": "Full announcement [here](https://medium.com/iearn/clarifying-2020-mergers-an-independent-iron-bank-a6f8f3f4c25e)."
   }
-}
+}
```

Comment:
nit: Quite a few newlines missing, probably worth to add them in

Reply:
it is an auto generated file from locsize now